docs/development.md

```diff
@@ -0,0 +1,142 @@
+ 
+ There are two type of React components, [presentational and container components](http://redux.js.org/docs/basics/UsageWithReact.html#presentational-and-container-components).
+ The presentational components ***are not aware of Redux***, and its main purpose is to define how the data is showed to the 
+ end reader. Meanwhile, the container components ***are aware of Redux*** and its main purpose if to keep the state and handle 
+ the logic of the app. 
+ 
+ #### Data flow 
+ 
+ The data flows in the following way:
+ 
+ 
+ 1. The user interact with the UI and he or she generates an action. The action is dispatched to the store via  `store.dispatch(action)`.
+ 2. The store handle the action using the reducer function we defined.
+ 3. The store saves the new state defined by the reducer in the previous step. 
+ 4. The UI is updated to reflect the last version of the state.
+ 
+ You can find more information about the data flow [here](http://redux.js.org/docs/basics/DataFlow.html)
+ ### Server side architecture
+ //TODO 
+
+ 
+## Project's structure
+
+ 
+ The client-side javascript sits inside the assets folder and it is organized in the following way:
+ 
+```
+ .
+ +-- assets
+ |   +-- components  // Shared components.
+ |   |   +-- exampleComponent1
+ |   |   |   +-- exampleComponent.jsx
+ |   |   |   +-- exampleComponent.css
+ |   |   +-- exampleComponent2
+ |   +-- helpers     // Shared helpers.
+ |   |   +-- helper.js  
+ |   +-- images      
+ |   +-- pages       // Support's pages
+ |   |   +-- page1   // An specific page.    
+ |   |   |   +-- actions    // Actions of a certain page/redux app.
+ |   |   |   +-- components // Components of a certain page/redux app.
+ |   |   |   |   +-- component1.jsx
+ |   |   |   |   +-- component2.jsx
+ |   |   |   +-- helpers    // Helpers of a certain page/redux app.
+ |   |   |   +-- reducers   // Reducers of a certain page/redux app. 
+ |   |   |   +-- page1.jsx  // Root component of a page.
+ |   |   |   +-- page1.scss // Stylesheet containing all the style of this page's components.
+ |   |   +-- page2
+ |   +-- stylesheets // Shared stylesheets
+ |       +-- main.scss //Entry point of the scss files.
+ ...
+```
+
+#### Important notes:
+
+* The UI of the project is organized in components. The shared components are self-contained and are located in the top
+ `components` folder. The components specific to a page and which are not used in other pages are located inside the 
+ `components` folder inside a specific page.
+ 
+* The CSS for a non-shareable component is located inside the `page.scss` file.  
+ 
+
+## Building process
+
+In order to build the project, team city runs a series of steps. The first step installs node js, the second build the 
+assets by executing the script [`build-tc`](https://github.com/guardian/support-frontend/blob/master/build-tc). 
+Finally, the third step builds the app by running the `compile` task define in 
```

Comment:
`Finally, the third step compiles the Scala code, packages the frontend and backend, and uploads this to riff-raff ready to be deployed`?